api/src/devcomp/infrastructure/datasources/learning-content/module.json

```diff
@@ -349,8 +348,8 @@
               "id": "8d7687c8-4a02-4d7e-bf6c-693a6d481c78",
               "type": "image",
               "url": "https://images.pix.fr/modulix/didacticiel/ordi-spatial.svg",
-              "alt": "",
-              "alternativeText": "Dessin d'un ordinateur dans un univers spatial."
+              "alt": "Dessin d'un ordinateur dans un univers spatial",
+              "alternativeText": ""
```

Comment:
Juste une remarque - est-ce qu'on est assez claire pour l'equipe contenu (et meme nous) pour la difference entre `alt` et `alternativeText`? peut-etre on a besoin de documenter quelque part?

Reply:
Yes good point, c'est documenté ici : https://1024pix.atlassian.net/wiki/spaces/CONT/pages/3255140365/In+pursuit+of+A11Y+aka+accessibilit#Description-des-images